virttest/qemu_storage.py

```diff
@@ -1516,12 +1527,24 @@ def amend(self, params, cache_mode=None, ignore_status=False):
                    if key.startswith('amend_')]
         if cache_mode:
             cmd_list.append("-t %s" % cache_mode)
+        secret_objects = self._secret_objects
+        if secret_objects:
+            # add a secret object, use for adding and erasing password
+            sec_id = params["secret_id"]
+            sec_data = params["secret_data"]
```

Comment:
These two params are used for amend only, can we add the amend_ prefix? and handle them before setting 'options', but the current 'amend_' params are used for options only, e.g. -o x=y,..., so @XueqiangWei @YongxueHong any other comments on this ?

Reply:
zhenchao, the amend_prefix is used for options only, so I didn't add the prefix to secret_id and secret_data. If adding the the amend_ prefix(e.g. amend_secret_id, amend_secret_data), as you said we need handle them before setting options and then pop them from params dict (e.g. del params[\"amend_secret_id\"], del params[\"amend_secret_data\"]). 

I think we can use the abbreviation, e.g. amnd_secret_id and amnd_secret_data. @zhencliu  @YongxueHong What's your opinion?

Reply:
@YongxueHong Please help review the above comments. Many thanks.

Reply:
Thank Yongxue and Zhenchao, I will update it.